ckanext/validation/plugin.py

```diff
@@ -188,10 +188,7 @@ def after_create(self, context, data_dict):
             for resource in data_dict.get(u'resources', []):
                 self._handle_validation_for_resource(context, resource)
         else:
-            # This is a resource. Resources don't need to be handled here
-            # as there is always a previous `package_update` call that will
-            # trigger the `before_update` and `after_update` hooks
-            pass
+            _run_async_validation(data_dict['id'])
```

Comment:
Does it cause the number of validation to be equal to N * N where N is number of resources in the dataset?